### DISCUSSION TITLE
"Session not authenticated" message for public permissioned app

### DISCUSSION THREAD

**Discussion by equ8r** [Budibase/budibase#6012] [Help] [answered]
I may be missing some nuance on how permissions work... but I have a few apps that are throwing 'Session Not Authenticated' errors when I try to navigate the app, even though all of the app screens are set to public. Anyone got an off-the-cuff explanation or a push toward what I might have done to cause it? I'm wanting to push the app on the local network and using device NAC to constrain access and I don't want to have to manage permissions for users.

For context - I'm running the latest version... in Docker on Windows 10, Postgres as the data source.

An aside... I've been using Budibase for about a week, and I am stunned at what a great open source project this is. Kudos to the people doing the heavy-lifting. I was looking for an alternative to power apps for a non-microsoft shop's narrow need and I discovered a powerful tool I suspect I'll use a lot moving ahead.

**Comment by joebudi** [comment, marked as answer]
Hey. Have you set the access role on the database to public? There's a setting above your tables called access.

**Comment by equ8r** [comment]
That's the kick I was looking for! That was it. Thanks a ton.

Love Budibase... a lot of features to assimilate!

**Comment by Sbleck** [comment]
Hello,

> Have you set the access role on the database to public? There's a setting above your tables called access.

Thanks for sharing this. Worked, as expected.
